azure-communication-ui/calling/src/main/java/com/azure/android/communication/ui/calling/CallComposite.java

```diff
@@ -670,7 +671,18 @@ public CallCompositeDebugInfo getDebugInfo(final Context context) {
      * @param context
      */
     public void bringToForeground(final Context context) {
-        showUI(context, false);
+        final boolean hasCallComposite = CallCompositeInstanceManager.hasCallComposite(instanceId);
```

Comment:
Shouldn't this fix go inside showUI?

Reply:
ShowUI is called from `launchComposite` too.